app/src/main/java/de/danoeh/antennapod/dialog/FeedPreferenceSkipDialog.java

```diff
@@ -0,0 +1,35 @@
+package de.danoeh.antennapod.dialog;
+
+import android.app.AlertDialog;
```

Comment:
Please use the `androidx` alert dialog. It has the same interface but it might have better compatibility with old Android versions.